tests/test_herseyland.py

```diff
@@ -0,0 +1,62 @@
+# mypy: allow-untyped-defs
+
+from recipe_scrapers.herseyland import HerseyLand
+from tests import ScraperTest
+
+
+class TestHerseyLandScraper(ScraperTest):
+
+    scraper_class = HerseyLand
+
+    def test_host(self):
+        self.assertEqual("hersheyland.com", self.harvester_class.host())
+
+    def test_author(self):
+        self.assertEqual("Herseyland", self.harvester_class.author())
+
+    def test_title(self):
+        self.assertEqual(
+            'HERSHEY\'S "Perfectly Chocolate" Chocolate Cake | Recipes',
```

Comment:
I was confused for a moment here about the use of single-quotes for the string (and the escaping).. but then noticed the double-quotes _within_ the string.  Makes sense :+1: